src/drive/web/modules/views/OnlyOffice/Toolbar/BackButton.jsx

```diff
@@ -4,13 +4,9 @@ import IconButton from 'cozy-ui/transpiled/react/IconButton'
 import Icon from 'cozy-ui/transpiled/react/Icon'
 import PreviousIcon from 'cozy-ui/transpiled/react/Icons/Previous'
 
-import { useRouter } from 'drive/lib/RouterContext'
-
-const BackButton = () => {
-  const { router } = useRouter()
-
+const BackButton = ({ onClick }) => {
   return (
-    <IconButton onClick={() => router.goBack()}>
+    <IconButton onClick={onClick} data-testid="onlyoffice-backButton">
       <Icon icon={PreviousIcon} />
     </IconButton>
   )
```

Comment:
backbutton should be useMemo() (to prevent re-render)

Reply:
J'ai pas compris l'idée.

Tu proposes que BackButton soit un composant mémoisé et celui-là exporté ?

Reply:
Ok compris, avec React.memo 👍